homeassistant/components/zone/__init__.py

```diff
@@ -37,27 +39,69 @@
 }, extra=vol.ALLOW_EXTRA)
 
 
+def create_zone(hass, name, latitude, longitude, radius, icon=None,
```

Comment:
Let's not create sync versions.

Reply:
I did that for two reasons. First, life360 is sync and needed it. Second, I saw it is very common for other code, even core code, to provide both async and sync versions. It makes sense to me to provide a proper sync version rather than relying on people writing sync code, who may or may not understand async, to do it correctly. Also, why force duplication of code? I guess ultimately it doesn't matter to me. If I remove it from here I'll just have to add it to life360.

Reply:
We only provide sync versions if it is an old API that was used in sync components. Life360 can just inline whatever this function does.